sormas-base/license-header

```diff
@@ -0,0 +1,14 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2020 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
```

Comment:
This will soon have to be changed to 2021 for modified files to have the correct header.